### PR TITLE
iperf: update page

### PR DESCRIPTION
**Pull request by einverne**
- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/iperf.md [hunk 1]
```diff
@@ -14,3 +14,15 @@
 - Run on client with 5 parallel threads:
 
 `iperf -c {{server_address}} -P {{5}}`
+
+- Run on client every 2 seconds:
+
+`iperf -c {{server_address}} -i {{2}}`
+
+- Run on server using UDP mode and set server port to listen on 5001:
```
Comment (zdroid) [inline pages/common/iperf.md:22]: Wouldn't it make more sense to switch this example and the one below?

Comment (einverne) [reply pages/common/iperf.md:22]: yeah, @zdroid  I agree.

Comment (zdroid) [reply pages/common/iperf.md:22]: Then the last example `- Run on server using UDP mode and set server port to listen on 5001:` should be moved up, no?

Comment (einverne) [reply pages/common/iperf.md]: @zdroid Take a look now.

**Comment by zdroid** [comment]
Now that's perfect!

**Pull request merged**